.github/workflows/ci.yml

```diff
@@ -58,13 +58,19 @@ jobs:
               run:
                 pip install .
 
+            - name: Install skyCatalogs
+              run: |
+                cd ..
+                git clone https://github.com/LSSTDESC/skyCatalogs.git
+
             - name: Install test deps
               run:
                 conda install -y pytest nose
 
             - name: Run tests
               run: |
+                setup -r ../skyCatalogs -j
```

Comment:
Any chance we can get skyCatalogs pip installable rather than using eups?  This is likely to be a blocker for some users down the line.  We don't want to force everyone to clone the repo to access it.

Reply:
I'll check with Joanne.  Should be doable.

Reply:
This line doesn't seem to have worked on the CI.  It's not able to import skyCatalogs currently.

Reply:
That's because skyCatalogs is trying to `import GCRCatalogs` at the top of the module.  I've already asked Joanne to move that import into the function that uses it so that it doesn't affect things here.

Reply:
That change was merged to master earlier this week.

Reply:
There is a file scripts/write_sed_files.py which still has an import of GCRCatalogs at the top level.
Could this be the problem?  I can similarly reorganize so that the import will be buried in a function if that would help.

Reply:
Yep.  Looks like it works now.  I restarted the tests and they passed.  Thanks Joanne!

Reply:
I don't think the imsim usage cares about what's in the scripts.

Reply:
Actually I think I can get rid of the import altogether.

Reply:
I've committed changes to skyCatalogs which eliminate that import